terraform/cloud_function/pubsub_triggered/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "bucket_name" {}
```

Comment:
Perhaps we can make this a bit more descriptive (like `source_code_bucket`) to make it clear what the purpose of this bucket is. Especially since we often pass different buckets to a function for different purposes